### DISCUSSION TITLE
7T DTI Data

### DISCUSSION THREAD

**Discussion by CristinaRoman1** [muscbridge/PyDKE#282] [Technical Support] [answered]
Hello PyDesigner Team, 

I was wondering if the PyDesigner pipeline has been tried on 7T diffusion data? Regardless, do you have any suggestions for optimizing our data? 

Thank you in advance for your time, 
Cristina

**Comment by TheJaeger** [comment]
Hi Cristina,

Is this for human or animal data?

**Comment by CristinaRoman1** [comment]
Hi Siddhartha,

This is for human data.

Thanks,
Cristina

On Tue, Jan 11, 2022 at 10:17 AM Siddhartha Dhiman ***@***.***>
wrote:

> Hi Cristina,
>
> Is this for human or animal data?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/m-ama/PyDesigner/discussions/282#discussioncomment-1948448>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQI3HZX22SARXLBWKOKOCGLUVRC2LANCNFSM5LUYJ5BA>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
*Cristina A. F. Rom**án, Ph.D. *

National MS Society Fellow

Neuropsychology and Neuroscience Lab

Kessler Foundation

Affiliation: Assistant Professor (Research), Department of Physical
Medicine & Rehabilitation and of Neurology, Rutgers New Jersey Medical
School

1199 Pleasant Valley Way

West Orange, NJ 07052
P. 973.965.6637
E: ***@***.***
Twitter: @CristinaAFRoman
pronouns: she/her(s)/ella

“*We cannot seek achievement for ourselves and forget about progress and
prosperity for our*

*community…Our ambitions must be broad enough to include the aspirations
and needs of others,*

*for their sakes and for our own*.”

-César Chávez

________________________________

*Confidentiality Notice*

The information in this email is intended *only for the individual or
entity to which it is addressed* and may contain information that is
privileged or confidential. If you are not the intended recipient, or the
employee or agent responsible for delivering the message to the intended
recipient, you are hereby notified that any dissemination, distribution or
copying of this communication is strictly prohibited, and you are requested
to please notify us immediately by telephone or email, and destroy the
original message.

**Reply by TheJaeger** [reply]
We don't have much experience with 7T human data - our lab primarily works with 3T for humans. I refer you to the paper [Diffusion MRI of the human brain at ultra-high field (UHF): A review](https://pubmed.ncbi.nlm.nih.gov/28428047/) that discusses strategies in optimizing 7T diffusion scans.

The computations and corrections don't change across main magnetic field, so you may still be able to use PyDesigner for those. Let us know if you require corrections currently present in PyDesigner, we'd be happy to include them in later versions.

**Comment by CristinaRoman1** [comment, marked as answer]
Thanks very much, Siddhartha. I sincerely appreciate your time. Perhaps I
can give it a go with PyDesigner and keep you posted?

One more quick question- how does your group generally use the QC metrics
that PyDesigner outputs? Are there any papers I can refer to for cutoffs,
how best to include in papers, etc?

Thanks again,
Cristina

On Tue, Jan 11, 2022 at 5:24 PM Siddhartha Dhiman ***@***.***>
wrote:

> We don't have much experience with 7T human data - our lab primarily works
> with 3T for humans. I refer you to the paper Diffusion MRI of the human
> brain at ultra-high field (UHF): A review
> <https://pubmed.ncbi.nlm.nih.gov/28428047/> that discusses strategies in
> optimizing 7T diffusion scans.
>
> The computations and corrections don't change across main magnetic field,
> so you may still be able to use PyDesigner for those. Let us know if you
> require corrections currently present in PyDesigner, we'd be happy to
> include them in later versions.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/m-ama/PyDesigner/discussions/282#discussioncomment-1950975>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQI3HZULVKA7IGMFCSWMDA3UVSU2VANCNFSM5LUYJ5BA>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
*Cristina A. F. Rom**án, Ph.D. *

National MS Society Fellow

Neuropsychology and Neuroscience Lab

Kessler Foundation

Affiliation: Assistant Professor (Research), Department of Physical
Medicine & Rehabilitation and of Neurology, Rutgers New Jersey Medical
School

1199 Pleasant Valley Way

West Orange, NJ 07052
P. 973.965.6637
E: ***@***.***
Twitter: @CristinaAFRoman
pronouns: she/her(s)/ella

“*We cannot seek achievement for ourselves and forget about progress and
prosperity for our*

*community…Our ambitions must be broad enough to include the aspirations
and needs of others,*

*for their sakes and for our own*.”

-César Chávez

________________________________

*Confidentiality Notice*

The information in this email is intended *only for the individual or
entity to which it is addressed* and may contain information that is
privileged or confidential. If you are not the intended recipient, or the
employee or agent responsible for delivering the message to the intended
recipient, you are hereby notified that any dissemination, distribution or
copying of this communication is strictly prohibited, and you are requested
to please notify us immediately by telephone or email, and destroy the
original message.

**Reply by TheJaeger** [reply]
We look at QC metrics when we observe anomalies in group-based stats. Suppose a subject's FA is relatively anomalous, it is then easy to look at SNR, outliers, and head motion to figure out whether the problem stems from acquisition or computations. There is another DTI QC pipeline that I highly recommend using: [PreQual](https://github.com/MASILab/PreQual)
